ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/MoneyMapper.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.cmc.ccd.mapper;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.domain.utils.MonetaryConversions;
+
+import java.math.BigDecimal;
+
+import static java.util.Objects.nonNull;
+import static org.apache.commons.lang3.StringUtils.isNotEmpty;
+
+@Component
```

Comment:
This doesnt need to be a component. Think of having them as mapper util static methods. So unnecessary injection in all the place can be avoided.

Reply:
why do we need A component for?

Reply:
These are not just utility method, these are changing to String representation as per CCD docs recommendation. It implements the consistent contract (interface `Mapper`) as other mapping components

Reply:
Bigdecimal to String or reverse doensnt need a component. It can be achieved by utility methods like how Java supports `Long.valueOf()` or similar. Component is overkill